framework/topology.h

```diff
@@ -15,6 +15,7 @@
 #include <string>
 #include <vector>
 
+#include <gettid.h>
```

Comment:
if you need `tid_t`, it's `<sys/types.h>`, not `<gettid.h>` (there's no standard header of that name).

Reply:
Good catch, I was meant to use `include "gettid.h"`. I'll fix it.

Reply:
It can't be sys/types.h because that won't define the type on Windows. We need our gettid.h